components/clp-py-utils/clp_py_utils/s3_utils.py

```
@@ -392,3 +384,134 @@ def _gen_deletion_config(objects_list: List[str]):
+    """
+    Gets the metadata of all objects under the <`bucket`>/<`key_prefix`>.
+
+    :param s3_client:
+    :param bucket:
+    :param key_prefix:
+    :return: A list of `FileMetadata` containing the object's metadata on success.
+    :raise: Propagates `boto3.client`'s exceptions.
+    :raise: Propagates `boto3.client.get_paginator`'s exceptions.
+    :raise: Propagates `boto3.paginator`'s exceptions.
+    """
+    file_metadata_list: List[FileMetadata] = list()
```

Ruff (0.14.0)

Unnecessary `list()` call (rewrite as a literal)

Rewrite as a literal

(C408)